### DISCUSSION TITLE
Delete Character Data by inactivity

### DISCUSSION THREAD

**Discussion by Dovakillz** [gameserverapp/Platform#853] [Suggestions]
Would be sweet to be able to delete character data based on X days of inactivity, kind of like Discords "Prune" option.

**Comment by Shiru94** [comment]
Who hasn't experienced this - players come and go, and keep creating new characters.

It would make manual deliveries and loading player bases much easier if there was a function that **automatically deleted player characters** that **hadn't been online for X days.**

This would not only make work easier, but would also avoid trouble with loyal players. This would make it less likely that:

▪︎players accidentally send packages to the wrong characters,

▪︎something is sent incorrectly when searching for players with similar names (from the past),

▪︎or the wrong tribe is loaded because a player is on vacation and has an old character.

▪︎Angry players can no longer accuse us of collecting player and tribe information.
